public/jsx/components/card-form.jsx

```diff
@@ -158,24 +164,20 @@ module.exports = React.createClass({
       // structure keyed by field or otherwise push onto
       // a list of generic errors.
       Object.keys(apiErrors).forEach(function(key) {
-        var errorList = apiErrors[key];
-        for (var i = 0; i < errorList.length; i += 1) {
-          var errorObj = errorCodes[errorList[i].code] || errorCodes.default;
-          errorObj.code = errorList[i].code;
-          if (errorObj.field) {
-            if (!newErrors.fields[errorObj.field]){
-              newErrors.fields[errorObj.field] = [];
-            }
-            newErrors.fields[errorObj.field].push(errorObj);
-            that.fieldProps[errorObj.field].showError = true;
-            that.fieldProps[errorObj.field].isValid = false;
-          } else {
-            newErrors.generic.push(errorObj);
-          }
+        console.log('API ErrorMessage: ' + JSON.stringify(apiErrors[key]));
+        var errorData = that.errorKeyToFieldMap[key] || {};
+        var field = errorData.field;
+        if (field) {
+          var fieldData = that.fieldProps[field];
+          fieldData.isValid = false;
+          fieldData.showError = true;
+          fieldData.errorMessage = fieldData.errors[errorData.error];
         }
```

Comment:
I was originally planning to fall-back to the larger form level error if this was the case. Because we wouldn't know what that realated to.